apiserver/facades/controller/charmrevisionupdater/testing/suite.go

```diff
@@ -76,7 +74,7 @@ type CharmSuite struct {
 	jcSuite *jujutesting.JujuConnSuite
 
 	charms map[string]*state.Charm
-	store  *mockStore
+	Store  *mockStore
```

Comment:
I don't like the fact that `charmrevisionupdater/testing` still exists, but it's used by `apiserver/facades/client/client/status.go:TestUpdateRevisions`. I'll take a look later to see if that test can be updated (or even just removed).

Reply:
I moved this into apiserver/facades/client/client where it's used.